<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerOCTReaderModule</class>
 <widget class="qSlicerWidget" name="qSlicerOCTReaderModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Module Template</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="selectFileButton">
          <property name="text">
           <string>Select File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QSpinBox" name="DimX">
          <property name="maximum">
           <number>4096</number>
          </property>
          <property name="value">
           <number>1024</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="DimY">
          <property name="maximum">
           <number>4096</number>
          </property>
          <property name="value">
           <number>1024</number>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="DimZ">
          <property name="maximum">
           <number>4096</number>
          </property>
          <property name="value">
           <number>256</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Dimensions:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Spacing:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="SpX">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="SpY">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QSpinBox" name="SpZ">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Scalar Type:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QComboBox" name="scalarTypeSelector">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <item>
           <property name="text">
            <string>Int 8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Unsigned Int 8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Int 16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Unsigned Int 16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Int 32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Unsigned Int 32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Float 32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Double 64</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="createVolumeButton">
          <property name="text">
           <string>Create Volume</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
